src/tests/unit/tests/DetailsView/components/assessment-instance-selected-button.test.tsx

```diff
@@ -80,11 +89,10 @@ describe('AssessmentInstanceSelectedButton', () => {
                     )
                     .verifiable(Times.once());
 
-                const wrapped = shallow(<AssessmentInstanceSelectedButton {...props} />);
+                const wrapped = render(<AssessmentInstanceSelectedButton {...props} />);
 
-                const iconButton = wrapped.find(IconButton);
-
-                iconButton.simulate('click', eventStub);
```

Comment:
This test and the following test can stay mostly the same by keeping `IconButton` mocked and calling `getMockComponentClassPropsForCall(IconButton).onClick(eventStub)`

Reply:
Hi
Made changes as per suggestions, please re-review.